
Header header
float64 exploredVolume
float64 timeConsumed